### ISSUE TITLE
implement_methods false positive - all methods abstract + alias

### ISSUE THREAD

**Issue by FreHu** [abaplint/abaplint#1410]
This is a nice combo

```abap
INTERFACES if_foo
  ALL METHODS ABSTRACT.
    
ALIASES:
 meth FOR if_foo~meth.
```

The aliased method does not need to be implemented here.

**Comment by larshp** [comment]
aliases 😢

**Comment by larshp** [comment]
first part, without aliases, take "ALL METHODS ABSTRACT" into account

```abap
INTERFACE lif_bar.
  METHODS m1.
ENDINTERFACE.

CLASS lcl_foo DEFINITION ABSTRACT.
  PUBLIC SECTION.
    INTERFACES lif_bar ALL METHODS ABSTRACT.
ENDCLASS.

CLASS lcl_foo IMPLEMENTATION.
ENDCLASS.
```

currently false positive,
![image](https://user-images.githubusercontent.com/5888506/94706579-a7085c80-0342-11eb-8459-8ee3c77b1a66.png)

**Comment by larshp** [comment]
try again, latest version

**Comment by FreHu** [comment]
seems to be fine even with the aliases now
